test/end-to-end/payrollConfig/payroll_config.spec.js

```diff
@@ -6,7 +6,7 @@ const chai = require('chai');
 /** configuring helpers* */
 helpers.configure(chai);
 
-describe('Payroll Configuration Management', () => {
+describe.only('Payroll Configuration Management', () => {
```

Comment:
I don't think we need the .only, could you remove it ?

Reply:
Whoops!  You are right.